spec/features/customers/show_spec.rb

```diff
@@ -0,0 +1,68 @@
+require "rails_helper"
+
+RSpec.describe "Customer Show",type: :feature do
+  before(:each) do
+    @customer_1 = Customer.create!(name: "Sally Shopper")
+
+    @supermarket_1 = Supermarket.create!(
+      name: "Corner Market",
+      location: "123 Food Street"
+    )
+
+    @banana = @supermarket_1.items.create!(
+      name: "Banana",
+      price: 1
+    )
+
+    @apple = @supermarket_1.items.create!(
+      name: "Apple",
+      price: 2
+    )
+
+    CustomerItem.create!(customer: @customer_1, item: @banana)
+    CustomerItem.create!(customer: @customer_1, item: @apple)
+  end
+  describe "as a visitor" do
+    describe "when I visit a customer show page" do
+      it "I see the customer's name, a list of its items including the item's name, price, and the name of the supermarket that it belongs to" do
+        visit "/customers/#{@customer_1.id}"
+
+        expect(page).to have_content("Sally Shopper")
+        
+        expect(page).to have_content("Items:")
```

Comment:
Remember that we can use `within` blocks to better test where things appear on our page.